src/main/java/teammates/common/util/Const.java

```diff
@@ -767,7 +767,7 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
         public static final String STUDENT_SHORT_NAME = "studentshortname";
         public static final String STUDENT_PROFILE_EMAIL = "studentprofileemail";
         public static final String STUDENT_PROFILE_INSTITUTION = "studentprofileinstitute";
-        public static final String STUDENT_NATIONALITY = "studentnationality";
+        public static final String STUDENT_NATIONALITY = "studentNationality";
```

Comment:
Take a look around this line: the standard for parameter names is all lowercase, so you should not break it.
Apologies for the ID; the actual naming convention is "with-hypens" which is very poorly enforced so neither case is right, so keep it as before for now.